fastlane/Fastfile

```diff
@@ -189,6 +189,8 @@ desc "Update Hybrid Common, Build PurchaseTester, and updates download scripts.
 lane :update_hybrid_common do |options|
   update_hybrid_common_versions(options)
   build_hybrid_example(options)
+  commit_current_changes(commit_message: 'Update examples')
+  push_to_git_remote(set_upstream: true)
```

Comment:
This pushes the changes made by `build_hybrid_example`. We were not pushing those changes to the branch

Reply:
Hmm this will push to the current branch, and if we call this from somewhere different than expected we might be coming to the wrong branch... Maybe we can check that we are not in main at least?

Reply:
I see, we do make a change to a bump branch in `update_hybrid_common_versions` but it's true that if we change anything in that lane, this could have unexpected consequences. 

I think it makes sense to check if we are in `main` yes.

I wish there was an easy way to do this at the same time we commit the version updates in  `update_hybrid_common_versions`, but that happens in `bump_phc_version` and I think it would be a big refactor